resources/views/app/feed.blade.php

```diff
@@ -0,0 +1,19 @@
+
+ @include('app.includes.overall.head')
+
+ <!-- Dropdown Structure -->
+@include('app.includes.overall.top_nav')
+
+
+
+
+<div class="row">
+    @include('app.includes.overall.side_nav')
+    @include('app.includes.feeds.body')
+</div>
+
+@include('app.includes.overall.footer')
+
+
+
+
```

Comment:
Rename this file to `index.blade.php`, Use `master` in place of `overall` . Reduce the space from line 5 to line 10. I need you to restructure this using `@yield` and `@extends`